src/portal/src/i18n/lang/fr-fr-lang.json

```diff
@@ -383,46 +383,46 @@
         "NEVER_EXPIRED": "Ne jamais expirer",
         "NAME_PREFIX": "Préfixe du nom du compte robot",
         "NAME_PREFIX_REQUIRED": "Le préfixe du nom du compte robot est obligatoire",
-        "UPDATE": "Update",
-        "AUDIT_LOG": "Audit Log",
-        "PREHEAT_INSTANCE": "Preheat Instance",
-        "PROJECT": "Project",
-        "REPLICATION_POLICY": "Replication Policy",
-        "REPLICATION": "Replication",
-        "REPLICATION_ADAPTER": "Replication Adapter",
-        "REGISTRY": "Registry",
-        "SCAN_ALL": "Scan All",
-        "SYSTEM_VOLUMES": "System Volumes",
-        "GARBAGE_COLLECTION": "Garbage Collection",
-        "PURGE_AUDIT": "Purge Audit",
+        "UPDATE": "Mettre à jour",
+        "AUDIT_LOG": "Log d'audit",
+        "PREHEAT_INSTANCE": "Préchauffer l'instance",
```

Comment:
```suggestion
        "PREHEAT_INSTANCE": "Précharger l'instance",
```

Reply:
I use the expression "préchauffage" to keep the translation consistent with other instances.

Reply:
Thank you for your suggestion.